islandora-connector-broadcast/src/main/java/ca/islandora/alpaca/connector/broadcast/EventRouter.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to Islandora Foundation under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * The Islandora Foundation licenses this file to you under the MIT License.
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://opensource.org/licenses/MIT
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package ca.islandora.alpaca.connector.broadcast;
+
+import org.apache.camel.builder.RouteBuilder;
+
+/**
+ * A content router distributing messages to multiple endpoints.
+ *
+ * @author Daniel Lamb
+ */
+public class EventRouter extends RouteBuilder {
```

Comment:
One thing that I've noticed when I generated the Javadocs for our Camel code is that I've got about a dozen `EventRouter` classes. You may want to think about naming this something like `BroadcastEventRouter` or similar.